Assets/Scripts/Cooldown.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Cooldown : MonoBehaviour
+{
+    [Header("DashBG")]
+    public Image dashCooldown;
+    private float cooldown = 3;
+    bool isCooldown = false;
+    public KeyCode dashCode;
```

Comment:
Public variables should be PascalCase, and grouped together above internal and private variables